Source/objects.cpp

```diff
@@ -4108,132 +4108,127 @@ void OperateLazStand(int pnum, int i)
 
 void OperateObject(int pnum, int i, BOOL TeleFlag)
 {
-	int v3;           // esi
-	int v4;           // edi
-	ObjectStruct *v5; // ebx
-	int v6;           // ecx
-	BOOLEAN sendmsg;  // [esp+Ch] [ebp-4h]
+	BOOL sendmsg;
 
-	v3 = pnum;
-	v4 = i;
-	sendmsg = pnum == myplr;
-	v5 = &object[i];
-	v6 = v5->_otype;
-	switch (v5->_otype) {
+	sendmsg = (pnum == myplr);
+	switch (object[i]._otype) {
 	case OBJ_L1LDOOR:
 	case OBJ_L1RDOOR:
 		if (TeleFlag) {
-			if (v6 == OBJ_L1LDOOR)
-				OperateL1LDoor(v3, i, OBJ_L1LDOOR);
-			if (v5->_otype == OBJ_L1RDOOR)
-				OperateL1RDoor(v3, v4, 1u);
-		} else if (v3 == myplr) {
-			OperateL1Door(v3, i, 1u);
+			if (object[i]._otype == OBJ_L1LDOOR)
+				OperateL1LDoor(pnum, i, OBJ_L1LDOOR);
+			if (object[i]._otype == OBJ_L1RDOOR)
+				OperateL1RDoor(pnum, i, 1u);

```

Comment:
```suggestion
				OperateL1RDoor(pnum, i, 1);
```

Same goes for other integer literals with `u` suffix.

Reply:
Actually it looks like these will turn into `TRUE` after cleaning up those functions but for now I will keep them as `1`.